Add missing changelog entry

tescande opened the pull request:
This patch adds a changelog entry for the noreplace flag added on the rsynd.conf config file and bumps the release version.

No other changes.

stormi commented:
After the merge, you can do a build to incoming as usual, and add the PR and build to the card that had the previous build (or create a new one, whatever you prefer).

tescande commented:
> After the merge, you can do a build to incoming as usual, and add the PR and build to the card that had the previous build (or create a new one, whatever you prefer).

I've updated the previous card with its own build info, moved it to "Ready for tests" and created a brand new one for this PR.
